### DISCUSSION TITLE
Get unpaid balance without actually pay

### DISCUSSION THREAD

**Discussion by ldricciardi** [coinfoundry/miningcore#1072] [Q&A]
Hello there, I am wondering if there is any way of getting unpaid balance of a miner but without paying them because I want to give miners the possibility of setting a custom payment threshold. But I suppose if I disable payment processing I wont be able to see the balances. Am I right?

I am thinking about not unlocking the etherbase address in geth. Could this be a solution?

**Comment by oliverw** [comment]
That's why the [MinimumPayout](https://github.com/oliverw/miningcore/blob/350d05fc973c0a3f8912b3be062d4235821c9384/src/Miningcore/Configuration/ClusterConfig.cs#L574) configuration option exists. It determines the minimum balance a miner must reach before a payment is issued. You could set this pretty high and allow the miner to change his personal threshold using the minerSettings API introduced in [v57](https://github.com/oliverw/miningcore/releases/tag/v57)

**Reply by ldricciardi** [reply]
Oh, I didn't see that changelog. Should I send a post request with "paymentthreshold" parameter, for instance, "paymentthreshold"=0.15 or I have to include other parameters?
Thank you in advance
